### PR TITLE
fixes #11171 - compute profiles nics & volumes select2 are initiated …

### PR DESCRIPTION
**Pull request by shlomizadok**
…separately

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -125,6 +125,8 @@ function onContentLoad(){
   $("#new_lookup_keys_ select").select2('destroy');
   // Interface selects are initiated separately
   $("#interfaces select").select2('destroy');
+  // Compute profiles additional nics & volumes are initiated separately
+  $(".nics_fields_template select, .volumes_fields_template select").select2('destroy');
```
Comment (shlomizadok) [inline app/assets/javascripts/application.js]: `$(".nics/volumes_fields_template select")` are using the same [code base in lookup_keys](https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/lookup_keys.js#L83), so they don't need to be initialized here

**Comment by domcleal** [comment]
What are your thoughts about @ares' comment here?  http://projects.theforeman.org/issues/11171#note-2

**Comment by shlomizadok** [comment]
My thought are that I'm gonna rename the title to wip :construction_worker: and change it

**Comment by shlomizadok** [comment]
2nd commit adds a css class that destoys select2 on those selects as they are initiated elsewhere.
If looks good, I'll squash and reword to include issue 11172

**Comment by ares** [comment]
now with the css class `without_select2`, would it be possible to initialize select2 only on selects without this class, rather than initialize everywhere and then destroy on specific ones? I think two jquery matches should do the trick using `:not` pseudoclass.

**Comment by shlomizadok** [comment]
@ares, so I thought the same... but [js] mystically it is not working with:

``` javascript```
$('select:not(".without_select2")').select2({ allowClear: true });
```

Not sure why :/

**Comment by ares** [comment]
@shlomizadok seems to work for me in Chrome and Firefox from web inspector, do you see any error in JS console?

**Comment by shlomizadok** [comment]
@ares, try adding a new interface on hosts#new, but without changing **any** other parameter (e.g., hosts -> new -> interfaces -> add interface)

**Comment by tbrisker** [comment]
@shlomizadok the extra quotes broke it:
`$('select:not(.without_select2)').select2({ allowClear: true });`
seems to work for me.

**Comment by domcleal** [comment]
Please note that this is still WoC and is needed for 1.9.0.

**Comment by ohadlevy** [comment]
afaik @shlomizadok is out until next week, feel free to pick it up if its
urgent.

**Comment by domcleal** [comment]
@tbrisker would you mind opening a new PR based on this with your suggested fix?

**Comment by tbrisker** [comment]
@domcleal created GH-2592

**Comment by domcleal** [comment]
Thanks @tbrisker!

**Pull request closed without merging**